conn.go

```diff
@@ -73,13 +73,21 @@ type Conn struct {
 	requiredAcks int32
 	apiVersions  map[apiKey]ApiVersion
 	fetchVersion apiVersion
+
+	transactionalId *string
 }
 
 // ConnConfig is a configuration object used to create new instances of Conn.
 type ConnConfig struct {
 	ClientID  string
 	Topic     string
 	Partition int
+
+	// The transactional id to use for transactional delivery. Idempotent
+	// deliver should be enabled if transactional id is configured.
+	// For more details look at transactional.id description here: http://kafka.apache.org/documentation.html#producerconfigs
+	// Empty string means that this connection can't be transactional.
+	TransactionalId string
```

Comment:
```suggestion
	TransactionalID string
```
This would match the spelling of `ClientID` and follow common Go naming conventions.